draft-amjad-cfrg-partially-blind-rsa.md

```diff
@@ -583,17 +583,6 @@ AugmentPublicKey in {{augment-public-key}}. In particular, an attacker can pick
 the output of AugmentPublicKey is very large, leading to more computational cost when verifying signatures.
 Thus, if attackers can force verification with metadata of their choosing, DoS attacks are possible.
 
-For applications where the values of potential public metadata choices are fixed ahead of time, it is possible
-to try and mitigate DoS attacks. If the set of possible metadata choices is small, then applications
-SHOULD use one of the protocol variants in {{RSABSSA}} with distinct keys for each metadata value.
```

Comment:
I think we should keep this advice. There's no strong reason to use this variant if there's few metadata choices.